torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -450,6 +450,9 @@ def rotate_image_tensor(
     fill: Optional[List[float]] = None,
     center: Optional[List[float]] = None,
 ) -> torch.Tensor:
+    if img.numel() == 0:
+        return img
```

Comment:
If expand is True, output can have larger size than the input. I mean output shape is changed, so probably we should do something like in bboxes: https://github.com/pytorch/vision/pull/6544/files#diff-8d715258c77dea35427ef8b4605c5dca131717b7e2f6ca1ee3afa99392813105R322

Reply:
Good catch!